testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeMinioDataLakeCaching.java

```diff
@@ -0,0 +1,100 @@
+import static io.trino.tests.product.launcher.env.common.Standard.CONTAINER_TRINO_CONFIG_PROPERTIES;
+import static io.trino.tests.product.launcher.env.common.Standard.CONTAINER_TRINO_ETC;
+import static io.trino.tests.product.launcher.env.common.Standard.CONTAINER_TRINO_JVM_CONFIG;
+import static io.trino.tests.product.launcher.env.common.Standard.createTrinoContainer;
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.utility.MountableFile.forHostPath;
+
+@TestsEnvironment
+public final class EnvMultinodeMinioDataLakeCaching
+        extends EnvironmentProvider
+{
+    private static final String CONTAINER_TRINO_DELTA_LAKE_PROPERTIES = CONTAINER_TRINO_ETC + "/catalog/delta.properties";
+    private static final String CONTAINER_TRINO_DELTA_LAKE_NON_CACHED_PROPERTIES = CONTAINER_TRINO_ETC + "/catalog/delta_non_cached.properties";
+    private final DockerFiles dockerFiles;
+    private final DockerFiles.ResourceProvider configDir;
+
+    private final String imagesVersion;
+    private final JdkProvider jdkProvider;
+    private final File serverPackage;
+    private final boolean debug;
+
+    @Inject
+    public EnvMultinodeMinioDataLakeCaching(
+            DockerFiles dockerFiles,
+            Standard standard,
+            Hadoop hadoop,
+            Minio minio,
+            EnvironmentConfig environmentConfig,
+            @ServerPackage File serverPackage,
+            JdkProvider jdkProvider,
+            @Debug boolean debug)
+    {
+        super(standard, hadoop, minio);
+        this.dockerFiles = requireNonNull(dockerFiles, "dockerFiles is null");
+        this.configDir = dockerFiles.getDockerFilesHostDirectory("conf/environment");
+        this.imagesVersion = environmentConfig.getImagesVersion();
+        this.jdkProvider = requireNonNull(jdkProvider, "jdkProvider is null");
+        this.serverPackage = requireNonNull(serverPackage, "serverPackage is null");
+        this.debug = debug;
+    }
+
+    @Override
+    public void extendEnvironment(Environment.Builder builder)
+    {
+        builder.configureContainer(COORDINATOR, container -> container
+                .withCopyFileToContainer(forHostPath(configDir.getPath("multinode/multinode-master-jvm.config")), CONTAINER_TRINO_JVM_CONFIG)
+                .withCopyFileToContainer(forHostPath(dockerFiles.getDockerFilesHostPath("common/standard-multinode/multinode-master-config.properties")), CONTAINER_TRINO_CONFIG_PROPERTIES)
+                .withTmpFs(ImmutableMap.of("/tmp/cache", "rw")));
+
+        builder.addConnector("delta_lake", forHostPath(configDir.getPath("multinode-minio-data-lake/delta.properties")), CONTAINER_TRINO_DELTA_LAKE_NON_CACHED_PROPERTIES);
+        builder.addConnector("delta_lake", forHostPath(configDir.getPath("multinode-minio-data-lake-cached/delta.properties")), CONTAINER_TRINO_DELTA_LAKE_PROPERTIES);
+
+        createTrinoWorker(builder, 0);
+        createTrinoWorker(builder, 1);
```

Comment:
I don't think you need to do this explicitly, just inject StandardMultinode in the same way as EnvMultinodeParquet